index.html

```diff
@@ -111,10 +111,13 @@
 
 <br>
 
-<div>
+<div style="height: 30px">
```

Comment:
elsewhere just uses <br> for spacing. This might be better but ideally change everywhere or use <br>